src/modm/platform/core/cortex/startup.c.in

```diff
@@ -190,5 +190,6 @@ Reset_Handler(void)
 	main();
 
 	modm_assert_debug(0, "core", "main", "exit");
-	while(1) ;
+	while (true)
+		{};
```

Comment:
No.

Reply:
`true` is not a keyword in C, you need `stdbool.h`. I don‘t agree that this change is an improvement.